### ISSUE TITLE
Fullscreen mode

### ISSUE THREAD

**Issue by ohcibi** [spolu/dwm.vim#29]
I'm not sure if I miss something but I looked very closely and found no implementation for Fullscreen mode. I just tried it out and it didnt work. So I looked into the code and found no mapping. I then looked for a function to map and found none either. I searched the Readme and the pull request if this is mentioned somewhere and still no luck. So I guess this is a feature request 8-)

**Comment by spolu** [comment]
Yep it existed before we changed the way buffers are moved around. So you are right this is a feature request! :)

**Comment by patricklewis** [comment]
Fullscreen mode as described by dwm is pretty much exactly what [ZoomWin](https://github.com/vim-scripts/ZoomWin) does, right?

**Comment by spolu** [comment]
Yep. That would be it.

**Comment by carlotrimarchi** [comment]
This means that it is normal that C-M doesn't do anything?

**Comment by spolu** [comment]
Yes C-M is not implemented at the moment. We should do something about it!

**Comment by nathankot** [comment]
Haven't looked into it, but https://github.com/szw/vim-maximizer relevant
